cli.js

```diff
@@ -8,17 +8,38 @@ const cli = meow(`
 	  $ slugify <string>
 
 	Options
-	  --separator=<string>  Word separator [Default: -]
+	  --separator=<string>                    Word separator [Default: -]
+	  --lowercase=<string>                    Make the slug lowercase. [Default: true]
+	  --decamelize=<string>                   Convert camelCase to separate words [Default: true]
+	  --preserve-leading-underscore=<string>  If your string starts with an underscore, it will be preserved in the slugified string. [Default: false]
 
 	Examples
 	  $ slugify 'Déjà Vu!'
 	  deja-vu
-	  $ slugify 'Unicorns & Rainbows' --separator='_'
+	  $ slugify 'Unicorns & Rainbows' --separator=_
 	  unicorns_and_rainbows
+	  $ slugify 'Déjà Vu!' --lowercase=false
+	  Deja-Vu
+	  $ slugify 'fooBar' --decamelize=false
+	  foobar
+	  $ slugify '_foo_bar' --preserve-leading-underscore=true
```

Comment:
Drop this one. We don't need examples for every option. This one is clear enough from the description.